src/config/settings.py

```diff
@@ -158,9 +160,14 @@ def set(
         self.dappnode = dappnode
 
         # hashi vault configuration
+        if hashi_vault_key_paths is not None:
+            if sorted(list(set(hashi_vault_key_paths))) != sorted(hashi_vault_key_paths):
```

Comment:
```suggestion
            if len(set(hashi_vault_key_paths)) != len(hashi_vault_key_paths):
```

Reply:
Updated in https://github.com/stakewise/v3-operator/pull/390/commits/515f0e1e6201bd8207969e8eabf9042b5a18eba6, thank you